pkg/pmem-csi-operator/pmem-tls/tls.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package pmemtls
```

Comment:
Why is there "pmem" in the package name? The package has nothing to do with PMEM...

The package also isn't specific to the operator. Why not simply "pkg/tls"?